### ISSUE TITLE
namespace associated with a feature is not being retained in the plugin

### ISSUE THREAD

**Issue by rwgdrummer** [ngageoint/geowave#664]
Set a default namespace for existing ingest formats.  https://github.com/ngageoint/geowave

The user has the option to set the namespace in the plugin config.  For some reason, it is not being used or was dropped.  Symptom is that the default 'gml' namespace is being used.

**Comment by rwgdrummer** [comment]
From an email:

Something is changing the name of a track feature to something other than "track"; causes GeoServer to return valid features.  For example, all attributes are returned when I change the feature type from "track" to "composite_track_points" and then reissue the request for tracks, but with a different typename.

geowave:8080/geoserver/ows?service=wfs&version=1.1.0&request=getfeature&typename=composite_track_points&maxfeatures=50&startIndex=0


Found the root of the issue.  For some reason, the geowave namespace is not being added to the features.  As a result, the features are given a default namespace www.opengis.net/gml.  This namespace includes the dynamicFeature.xsd, which already has a track feature type.
schemas.opengis.net/gml/3.1.1/base/dynamicFeature.xsd.

I added a fix for this issue a while ago, but something has changed between 0.8.9 and 0.9.0 in this area.
